docs/course/index.md

```diff
@@ -130,11 +130,7 @@ This file specifies basic information about the course:
     }
     ```
 
-??? note "Format specification for `infoCourse.json`"
-
-    ```json
-    --8<-- "apps/prairielearn/src/schemas/schemas/infoCourse.json"
-    ```
+!!! info "[`infoCourse.json` reference](../schemas/infoCourse.md) :robot:"
```

Comment:
Can we force something to make links into actual links?

<img width="820" alt="Screenshot 2025-03-27 at 14 18 42" src="https://github.com/user-attachments/assets/f0c12803-540e-4f27-a672-11477eaf3c1b" />

Reply:
Yes -- You add angle brackets. Should I do that in this PR, or a separate one?

Reply:
This PR is fine.